src/test/java/com/orbitz/consul/BaseIntegrationTest.java

```diff
@@ -9,10 +9,12 @@
 
     protected static Consul client;
 
+    protected static HostAndPort clientAddress = HostAndPort.fromParts("localhost", 8500);
```

Comment:
Since this field is hidden from actual test implementations, it seems to be that this name is not explanatory enough. 

I'd prefer something like `defaultClientAddress` or `defaultHostAndPort` (since `hostAndPport` is the builder method name, that seems to make more sense).

Reply:
Done.